receiver/kubeletstatsreceiver/internal/kubelet/volume_test.go

```diff
@@ -148,6 +148,39 @@ func TestDetailedPVCLabels(t *testing.T) {
 				"k8s.namespace.name":             "pod-namespace",
 			},
 		},
+		{
+			name:       "persistentVolumeClaim - with detailed PVC labels (CSI PV with GCP driver)",
```

Comment:
Could we also enhance the scraper's tests like at https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/receiver/kubeletstatsreceiver/scraper_test.go#L598? The resulted files are really useful for reference since they contain the whole information.